### PR TITLE
css-two-column-layout

### PR DESCRIPTION
**Pull request by Alex-piper**
Screen Shot 2022-10-16 at 8 39 05 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  margin-left: 325px;
+  margin-top: 35px;
+}
+
+p {
+  line-height: 2;
+  font-size: 12px;
+}
+
+.one-half {
```
Comment (Zhiwutian) [inline css-two-column-layout/styles.css:57]: make sure to have a proper flex basis for this class. We always want to explicitly set a column width

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+.row {
+  display: flex;
+  width: 100%;
+  flex-wrap: nowrap;
+  height: 160px;
+  padding-top: 50px;
+}
+
+.four-fifths {
```
Comment (Zhiwutian) [inline css-two-column-layout/styles.css:35]: make sure to include a proper flex basis here

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        </div>
+        <div class="four-fifths">
+          <section>
+            <h3>For any circumstance</h3>
+            <p>Our intuitive system is built around the peculiarities of your facility; we support offline deployments, sync to the cloud and run on any device 24/7.</p>
+          </section>
+        </div>
+      </div class="one-half">
+      <div class="one-half">
+        <div class="one-fifth">
```
Comment (Zhiwutian) [inline css-two-column-layout/index.html:55]: make sure to include a `one-fifth` class in the CSS with a proper flex basis assigned to it.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great Alex, just a couple small css updates!

**Comment by Alex-piper** [comment]
Screen Shot 2022-10-23 at 8 10 49 PM (2)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**